upup/models/cloudup/resources/addons/podsecuritypolicy.addons.k8s.io/k8s-1.9.yaml.template

```diff
@@ -55,6 +55,12 @@ subjects:
 - kind: User
   name: kubelet
   apiGroup: rbac.authorization.k8s.io
+## TODO: need to question whether this can move into a rolebinding?
```

Comment:
If possible we should do whatever upstream does.  I think it does use a rolebinding:

cluster/gce/addons/podsecuritypolicies/node-binding.yaml